### PR TITLE
Minor fixes in docs and doc's code-style

### PR DESCRIPTION
**Pull request by vishesh92**
Minor documentation fixes and use 79  as max line length in docs.

### CODE REVIEW

File: docs/source/deploy.rst [hunk 1]
```diff
@@ -188,8 +188,8 @@ Another useful tool to monitor and control Gunicorn is Supervisor_. A
 
 Upstart
 -------
-Using Gunicorn with upstart is simple. In this example we will run the app "myapp"
-from a virtualenv. All errors will go to /var/log/upstart/myapp.log.
+Using Gunicorn with upstart is simple. In this example we will run the app
+"myapp" from a virtualenv. All errors will go to /var/log/upstart/myapp.log.
```
Comment (berkerpeksag) [inline docs/source/deploy.rst]: \`\`/var/log/upstart/myapp.log\`\`

File: docs/source/deploy.rst [hunk 2]
```diff
@@ -276,9 +276,12 @@ utility::
 
     kill -USR1 $(cat /var/run/gunicorn.pid)
 
-.. note:: overriding the LOGGING dictionary requires to set `disable_existing_loggers: False`` to not interfere with the Gunicorn logging.
+.. note:: overriding the LOGGING dictionary requires to set
```
Comment (berkerpeksag) [inline docs/source/deploy.rst]: overriding -> Overriding

Comment (berkerpeksag) [inline docs/source/deploy.rst]: LOGGING -> \`\`LOGGING\`\`

Comment (berkerpeksag) [inline docs/source/deploy.rst]: Also, please use the following format:

```rst
.. note::
   Overriding the ``LOGGING`` ...
```

File: docs/source/deploy.rst [hunk 3]
```diff
@@ -276,9 +276,12 @@ utility::
...
+   ``disable_existing_loggers: False`` to not interfere with the Gunicorn
+   logging.
 
-.. warning:: Gunicorn error log is here to log errors from Gunicorn, not from another application.
+.. warning:: Gunicorn error log is here to log errors from Gunicorn, not from
```
Comment (berkerpeksag) [inline docs/source/deploy.rst]:
```rst
.. warning::
   Gunicorn error log ...
```

File: docs/source/design.rst [hunk 4]
```diff
@@ -67,16 +67,16 @@ Choosing a Worker Type
...
 The default synchronous workers assume that your application is resource-bound
 in terms of CPU and network bandwidth. Generally this means that your
-application shouldn't do anything that takes an undefined amount of time. An example
-of something that takes an undefined amount of time is a request to the internet.
-At some point the external network will fail in such a way that clients will pile up on your
-servers. So, in this sense, any web application which makes outgoing requests to
-APIs will benefit from an asynchronous worker.
+application shouldn't do anything that takes an undefined amount of time. An
+example of something that takes an undefined amount of time is a request to the
+internet.At some point the external network will fail in such a way that
```
Comment (berkerpeksag) [inline docs/source/design.rst]: Add a space after ".".

File: docs/source/design.rst [hunk 5]
```diff
@@ -119,12 +119,12 @@ system, using multiple threads, multiple worker processes, or some mixture, may
...
-as the application code will be shared among workers but loaded only in the
-worker processes (unlike when using the preload setting, which loads the code in
-the master process).
+Gunicorn, while still allowing for application upgrades using the reload
+signal, as the application code will be shared among workers but loaded only in
+the worker processes (unlike when using the preload setting, which loads the
+code in the master process).
 
-.. note:: Under Python 2.x, you need to install the 'futures' package to use
+.. note:: Under Python 2.x, you need to install the `futures` package to use
```
Comment (berkerpeksag) [inline docs/source/design.rst]: Please revert this. \`foo\` is not a best practice.

File: docs/source/instrumentation.rst [hunk 6]
```diff
@@ -13,7 +13,8 @@ The use of UDP cleanly isolates Gunicorn from the receiving end of the statsD
 metrics so that instrumentation does not cause Gunicorn to be held up by a slow
 statsD consumer.
 
-To use statsD, just tell Gunicorn where the statsD server is:
+To use statsD, add ``statsd_host = 'localhost:8125'`` in configuration file or
+just tell Gunicorn where the statsD server is:
```
Comment (berkerpeksag) [inline docs/source/instrumentation.rst]: Please revert this too. The documentation is already good as is.

You can add a link to http://docs.gunicorn.org/en/stable/settings.html#statsd-host if you want to.

File: docs/source/settings.rst [hunk 7]
```diff
@@ -82,7 +82,7 @@ You'll want to vary this a bit to find the best for your particular
 application's work load.
 
 By default, the value of the ``WEB_CONCURRENCY`` environment variable.
-If it is not defined, the default is 1.
+If it is not defined, the default is ``1``.
```
Comment (berkerpeksag) [inline docs/source/settings.rst]: This file is automatically generated. Please update `gunicorn/config.py` instead.

If you didn't touch this file and it was updated by `make html`, please revert all changes to it. I can do it in a separate commit.

File: docs/source/signals.rst [hunk 8]
```diff
@@ -89,14 +89,17 @@ incoming requests together. To phase the old instance out, you have to
...
 processes will start to gracefully shut down.
 
-At this point you can still revert to the old server because it hasn't closed its listen sockets yet, by following these steps:
+At this point you can still revert to the old server because it hasn't closed
+its listen sockets yet, by following these steps:
 
-- Send the HUP signal to the old master process - it will start the worker processes without reloading a configuration file
-- Send the TERM signal to the new master process to gracefully shut down its worker processes
+- Send the HUP signal to the old master process - it will start the worker
+   processes without reloading a configuration file
```
Comment (berkerpeksag) [inline docs/source/signals.rst]: Broken indentation.

```rst
- Send the HUP signal ...
  processes ...
```

File: docs/source/signals.rst [hunk 9]
```diff
@@ -89,14 +89,17 @@ incoming requests together. To phase the old instance out, you have to
...
+- Send the TERM signal to the new master process to gracefully shut down its
```
Comment (berkerpeksag) [inline docs/source/signals.rst]: Broken indentation.

```rst
- Send the TERM signal ...
  worker processes
```

**Comment by vishesh92** [comment]
@berkerpeksag Will make the changes in a bit and update here. Thanks.

**Comment by vishesh92** [comment]
@berkerpeksag Just updated the PR. Please check.

**Comment by tilgovi** [comment]
Looks great. Needs a rebase.

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**